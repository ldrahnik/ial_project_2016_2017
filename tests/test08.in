A -5 B
B 4 C
C 1 D
A 3 D
A 2 C
