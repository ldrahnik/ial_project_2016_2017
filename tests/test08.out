FloydWarshall: A -(-5)> B -(-1)> C -(0)> D
BellmanFord: A -(-5)> B -(-1)> C -(0)> D
Johnson: A -(0)> B -(0)> C -(0)> D
